payments-app-extension-credit-card/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,24 @@
+[[extensions]]
+name = "{{ name }}"
+type = "payments_app_extension"
+handle = "{{ handle }}"
+target = "credit_card"
+
+merchant_label = "Credit Credit Payments App Extension"
+start_payment_session_url = "https://api.paymentprovider.com/endpoint"
+start_refund_session_url = "https://api.paymentprovider.com/refund"
+start_capture_session_url = "https://api.paymentprovider.com/capture"
+start_void_session_url = "https://api.paymentprovider.com/void"
+start_verification_session_url = "https://api.paymentprovider.com/verification"
+supported_countries = ["US"]
+supported_payment_methods = ["visa"]
+supports_3ds = false
+supports_installments = false
+supports_deferred_payments = false
```

Comment:
Seems like the UI groups these two together. Is there a problem with that?

Reply:
yes, good point - i think we might need to introduce validation in CLI or partners for instance